docs/source/conf.py

```diff
@@ -0,0 +1,340 @@
+htmlhelp_basename = 'Pibooth docs'
+
+# -- Options for LaTeX output ---------------------------------------------
+latex_engine = 'pdflatex'
+
+latex_elements = {
+        # The paper size ('letterpaper' or 'a4paper').
+        'papersize'        : 'a4paper',
+        # The font size ('10pt', '11pt' or '12pt').
+        'pointsize'        : '12pt',
+        # Latex figure (float) alignment
+        'figure_align'     : 'htbp',
+        'extraclassoptions': 'openany',
+        'preamble'         : r'''
+        %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+        %%%add number to subsubsection 2=subsection, 3=subsubsection
+        \setcounter{secnumdepth}{0}
+        %%%% Table of content upto 2=subsection, 3=subsubsection
+        \setcounter{tocdepth}{2}
+    ''',
+
+        'sphinxsetup'      : \
+            'hmargin={0.7in,0.7in}, vmargin={0.7in,0.7in}, \
+        marginpar=1in, \
+        verbatimwithframe=False, \
+        TitleColor={rgb}{0,0,0}, \
+        HeaderFamily=\\rmfamily\\bfseries, \
+        InnerLinkColor={rgb}{0,0,1}, \
+        OuterLinkColor={rgb}{0,0,1}',
+}
+
+# Grouping the document tree into LaTeX files. List of tuples
+# (source start file, target name, title,
+#  author, documentclass [howto, manual, or own class]).
+
+latex_documents = [
+        ('index', 'Pibooth.tex', 'Documentation',
+         latex_authors, 'manual', True),
+]
+
+# The name of an image file (relative to this directory) to place at the top of
+# the title page.
+latex_logo = './SupplementaryFiles/pibooth.png'
+
+# For "manual" documents, if this is true, then toplevel headings are parts,
+# not chapters.
+latex_use_parts = False
+
+# If true, show page references after internal links.
+# latex_show_pagerefs = False
+
+# If true, show URL addresses after external links.
+# latex_show_urls = False
+
+# Documents to append as an appendix to all manuals.
+# latex_appendices = []
+
+# If false, no module index is generated.
+# latex_domain_indices = True
+
+latex_toplevel_sectioning = 'section'
+
+# -- Options for manual page output ---------------------------------------
+
+# One entry per manual page. List of tuples
+# (source start file, name, description, authors, manual section).
+man_pages = [
+
+]
+
+# If true, show URL addresses after external links.
+# man_show_urls = False
+
+
+# -- Options for Texinfo output -------------------------------------------
+
+# Grouping the document tree into Texinfo files. List of tuples
+# (source start file, target name, title, author,
+#  dir menu entry, description, category)
+texinfo_documents = [
+
+]
+
+
+# Documents to append as an appendix to all manuals.
+# texinfo_appendices = []
+
+# If false, no module index is generated.
+# texinfo_domain_indices = True
+
+# How to display URL addresses: 'footnote', 'no', or 'inline'.
+# texinfo_show_urls = 'footnote'
+
+# If true, do not generate a @detailmenu in the "Top" node's menu.
+# texinfo_no_detailmenu = False
+
+def setup(app):
```

Comment:
Why you are not using the `html_style` ?
The original css from readthedoc theme seems ok to me.

Reply:
I use old project as template, bad idea...